plugins/package-managers/gradle-plugin/src/main/kotlin/GradleModelExtensions.kt

```diff
@@ -51,9 +50,3 @@ internal fun Configuration.isRelevant(): Boolean {
 
     return canBeResolved && !isDeprecatedConfiguration && !isDependenciesMetadata
 }
-
-/**
- * Return whether the Android plugin for Gradle has been applied to the project.
- */
-internal fun Project.isAndroidProject(): Boolean =
```

Comment:
I originally wanted to keep this around as I was planning to take it into use again, but as I don't know when that will be I'm fine with removing it. I know how to restore the implementation in any case.